contracts/core/Registry.sol

```diff
@@ -124,7 +124,7 @@ contract Registry is IRegistry, Native, AccessControl, Transfer, Initializable,
         address[] memory _members
     ) external returns (bytes32) {
         // Generate a profile ID using a nonce and the msg.sender
-        bytes32 profileId = _generateProfileId(_nonce);
+        bytes32 profileId = _generateProfileId(_nonce, _owner);
```

Comment:
Worth noting:

- A profile ID is not regenerated when the owner changes
- This means if a profile owner changes, while the new profile owner can recreate the same profile anchor on a new chain, they will not be able to control the profile as only the owner (aka the old owner) would have access to it.

This exists in our current architecture as well as the msg.sender has to be the same across chains to ensure your project has the same profile Id and anchor address
In that sense, this is a more logical step as it give you the ability to create profiles on behalf of someone else but not do anything beyond that